### PR TITLE
Fix Gradle config instructions

### LINKED ISSUE #1038: NullAway configuration is incorrect
Hello,

I've tried to search for the simillar issue but haven't found anything yet.
When trying to build gradle project with Nullaway I'm having this exception:
```
An exception has occurred in the compiler (21.0.2). Please file a bug against the Java compiler via the Java bug reporting page  (https://bugreport.java.com) after checking the Bug Database (https://bugs.java.com) for duplicates. Include your program, the following   diagnostic, and the parameters passed to the Java compiler in your report. Thank you.
java.lang.AssertionError: com.google.errorprone.scanner.ErrorProneInjector$ProvisionException: Failed to initialize com.uber.nullaway.NullAway
	...
Caused by: java.lang.IllegalStateException: DO NOT report an issue to Error Prone for this crash!  NullAway configuration is incorrect.  Must specify annotated packages, using the -XepOpt:NullAway:AnnotatedPackages=[...] flag.  If you feel you have gotten this message in error report an issue at https://github.com/uber/NullAway/issues.
	at com.uber.nullaway.ErrorProneCLIFlagsConfig.<init>(ErrorProneCLIFlagsConfig.java:169)
	at com.uber.nullaway.NullAway.<init>(NullAway.java:294)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
```

**build.gradle configuration:**
```
plugins {
    id 'com.gradleup.shadow' version '8.3.1'
    id 'java'
    id 'org.springframework.boot' version '3.3.3'
    id 'io.spring.dependency-management' version '1.1.6'
    id "net.ltgt.errorprone" version "4.0.1"
}

group = 'helium.video'
version = '0.0.1-SNAPSHOT'

java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(21)
    }
}

repositories {
    mavenCentral()
}

dependencies {
    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'

    implementation 'org.springframework.boot:spring-boot-starter-web'
    implementation('org.springframework.boot:spring-boot-starter-web') {
        exclude group: 'org.springframework.boot', module: 'spring-boot-starter-logging'
    }
    implementation 'org.springframework.boot:spring-boot-starter-log4j2'

    implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
    implementation 'org.springframework.boot:spring-boot-starter-validation'
    runtimeOnly 'com.mysql:mysql-connector-j'
    testImplementation 'org.springframework.boot:spring-boot-starter-test'

    // TODO: add spring security

    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'

    errorprone 'com.uber.nullaway:nullaway:0.10.25'
    compileOnly "com.google.code.findbugs:jsr305:3.0.2"
    errorprone 'com.google.errorprone:error_prone_core:2.32.0'
    errorproneJavac "com.google.errorprone:javac:9+181-r4173-1"

}

import net.ltgt.gradle.errorprone.CheckSeverity

tasks.withType(JavaCompile) {
    // remove the if condition if you want to run NullAway on test code
    if (!name.toLowerCase().contains("test")) {
        options.errorprone {
            check("NullAway", CheckSeverity.WARN)
            option("NullAway:AnnotatedPackages", "helium.video")
        }
    }
}

tasks.named('test') {
    useJUnitPlatform()
}
```

I just don't understand what the issue with `annotating packages` when I'm using this option `option("NullAway:AnnotatedPackages", "helium.video")`.

```
JDK: 21
nullaway: 0.10.25
erorr_prone_core: 2.32.0
```

### PR DESCRIPTION
**Pull request by msridhar**
Fixes #1038

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +52,7 @@ Let's walk through this script step by step.  The `plugins` section pulls in the
...
...n options to NullAway.  First `check("NullAway", CheckSeverity.ERROR)` sets NullAway issues to the error level (it's equivalent to the `-Xep:NullAway:ERROR` standard Error Prone argument); by default NullAway emits warnings.  Then, `option("NullAway:AnnotatedPackages", "com.uber")` (equivalent to the `-XepOpt:NullAway:AnnotatedPackages=com.uber` standard Error Prone argument) tells NullAway that source code in packages under the `com.uber` namespace should be checked for null dereferences and proper usage of `@Nullable` annotations, and that class files in these packages should be assumed to have correct usage of `@Nullable` (see [the docs](https://github.com/uber/NullAway/wiki/Configuration) for more detail).  NullAway requires at least the `AnnotatedPackages` configuration argument to run, in order to distinguish between annotated and unannotated code.  See [the configuration docs](https://github.com/uber/NullAway/wiki/Configuration) for other useful configuration options.  For even simpler configuration of NullAway options, use the [Gradle NullAway plugin](https://github.com/tbroyer/gradle-nullaway-plugin).  Finally, we give show how to disable NullAway on test code, if desired.
```
Comment (yuxincs) [inline README.md:55]:
```suggestion
Finally, in the `tasks.withType(JavaCompile)` section, we pass some configuration options to NullAway.  First `check("NullAway", CheckSeverity.ERROR)` sets NullAway issues to the error level (it's equivalent to the `-Xep:NullAway:ERROR` standard Error Prone argument); by default NullAway emits warnings.  Then, `option("NullAway:AnnotatedPackages", "com.uber")` (equivalent to the `-XepOpt:NullAway:AnnotatedPackages=com.uber` standard Error Prone argument) tells NullAway that source code in packages under the `com.uber` namespace should be checked for null dereferences and proper usage of `@Nullable` annotations, and that class files in these packages should be assumed to have correct usage of `@Nullable` (see [the docs](https://github.com/uber/NullAway/wiki/Configuration) for more detail).  NullAway requires at least the `AnnotatedPackages` configuration argument to run, in order to distinguish between annotated and unannotated code.  See [the configuration docs](https://github.com/uber/NullAway/wiki/Configuration) for other useful configuration options.  For even simpler configuration of NullAway options, use the [Gradle NullAway plugin](https://github.com/tbroyer/gradle-nullaway-plugin). Finally, we show how to disable NullAway on test code, if desired.
```

**Review by yuxincs — APPROVED** [review]
Typo fix suggestion, but otherwise LGTM :)

File: README.md (hunk 1 above)
Comment (msridhar) [reply README.md:55]: Sorry the PR was set on auto-merge!  Also somehow GitHub is not highlighting your typo fix and I can't find what you changed 😐

Comment (yuxincs) [reply README.md:55]: It's the last line `Finally, we show how to disable NullAway on test code, if desired.` :) but not a big deal of course!

**Pull request merged**